mne/channels/montage.py

```diff
@@ -463,14 +482,30 @@ def __init__(self, hsp=None, hpi=None, elp=None, point_names=None,
         self.nasion = nasion
         self.lpa = lpa
         self.rpa = rpa
-        self.dev_head_t = dev_head_t
         self.dig_ch_pos = dig_ch_pos
         if not isinstance(coord_frame, str) or \
                 coord_frame not in _str_to_frame:
             raise ValueError('coord_frame must be one of %s, got %s'
                              % (sorted(_str_to_frame.keys()), coord_frame))
         self.coord_frame = coord_frame
 
+        self.dev_head_t = dev_head_t
+
+        # XXX: I'm having second thoughts on if we should represent the data
+        #      as a list of dicts, or keep each structure and create the list
+        #      of dicts as it was (but with a getter property)
+        #
+        #      If DigMontage is just constructed and nothing else (maybe some
+        #      plotting helper, or whatever) then `list_of_dicts` is fine. But
+        #      if we have to allow any operation to the points (i.e: rotation,
+        #      scale, change frame of reference, whatever) then it would be
+        #      really complicated.
+
+        self.dig = _make_dig_points(
+            nasion=self.nasion, lpa=self.lpa, rpa=self.rpa, hpi=self.elp,
+            extra_points=self.hsp, dig_ch_pos=self.dig_ch_pos
```

Comment:
now that this is done @massich I would expose nasion, lpa, rpa, elp, hsp and dig_ch_pos as deprecated properties computed from `self.dig` so we move towards a object fully defined by the dig attribute and the remaining point_names. Now I find puzzling that `_make_dig_points` does not take `coord_frame` as input. I assume it assumes that all the points are in head coordinate system.